EIPS/eip-1.md

```diff
@@ -1,101 +1,100 @@
-  - **ERC** - application-level standards and conventions, including contract standards such as token standards ([ERC20]), name registries ([ERC26], [ERC137]), URI schemes ([ERC67]), library/package formats ([EIP82]), and wallet formats ([EIP75], [EIP85]).
-- An **Informational EIP** describes an Ethereum design issue, or provides general guidelines or information to the Ethereum community, but does not propose a new feature. Informational EIPs do not necessarily represent Ethereum community consensus or a recommendation, so users and implementers are free to ignore Informational EIPs or follow their advice.
-- A **Meta EIP** describes a process surrounding Ethereum or proposes a change to (or an event in) a process. Process EIPs are like Standards Track EIPs but apply to areas other than the Ethereum protocol itself. They may propose an implementation, but not to Ethereum's codebase; they often require community consensus; unlike Informational EIPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Ethereum development. Any meta-EIP is also considered a Process EIP.
+- A **Standard Track EEP** describes any change that affects most or all EOS implementations, such as a change to the the network protocol, a change in block or transaction validity rules, proposed application standards/conventions, or any change or addition that affects the interoperability of applications using EOS. Furthermore Standard EEPs can be broken down into the following categories. Standards Track EEPs consist of three parts, a design document, implementation, and finally if warranted an update to the [formal specification].
+  - **Core** - improvements requiring a consensus fork, as well as changes that are not necessarily consensus critical but may be relevant to [“core dev” discussions] [FLAG - DO WE ALSO HAVE CONSENSUS FORKS ON EOS?] [FLAG - WE DON'T HAVE CORE DEVS, SHOULD THAT BE REPLACED BY SOMETHING?]
+  - **Networking** - includes improvements around [devp2p] and [Light Ethereum Subprotocol], as well as proposed improvements to network protocol specifications of [whisper] and [swarm]. [FLAG - Do these make sense/do we have a similar thing? [devp2p] and [Light Ethereum Subprotocol] and  specifications of [whisper] and [swarm]]
+  - **Interface** - includes improvements around client [API/RPC] specifications and standards, and also certain language-level standards like method names and [contract ABIs]. The label “interface” aligns with the [FLAG - DO WE NEED TO CHANCE interfaces repo] [interfaces repo] and discussion should primarily occur in that repository before an EEP is submitted to the EEPs repository.
```

Comment:
I don't think there is something as similar as the Ethereum interaces repo for EOS. However, we could use this wording IMO:

```
includes improvements around client [API] specifications and standards, and also certain language-level standards like [contract ABIs]. For [contract ABIS], it aligns with the [eosio.contracts repo] and discussion should primarily occur in that repository before an EEP is submitted to the EEPs repository
```

That would link to `https://github.com/EOSIO/eosio.contracts`

Reply:
Also, if we have some links for the EOS platform API, that could be great.